Update KEB swagger with OSB schema (#636)
kyma-project/control-plane: docs/kyma-environment-broker/03-11-swagger.md

KEB uses a Swagger schema file that is mounted as a volume to the Pod. You can find that schema [here](https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-environment-broker/files/swagger.yaml).

> You can use templates in the Swagger schema file to parametrize it.

Reviewer: ```suggestion
You can use templates in the Swagger schema file to configure it.
```

Reviewer: Please join this sentence with the previous one.

   https://$BROKER_URL/
   ```

> With this option you need the browser [extension](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf) to use the `Try it out` feature.

Reviewer: ```suggestion
To use the `Try it out` feature on Virtual Service, you need this [Chrome browser extension](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf).
```
